ilastik/applets/tracking/base/trackingBaseDataExportApplet.py

```diff
@@ -185,4 +216,75 @@ def _configure_operator_with_parsed_args(cls, parsed_args, opTrackingDataExport)
 
+    def _export_with_plugin(self, lane_index: int, checkOverwriteFiles: bool, pluginName: str) -> bool:
+        argsSlot = self.topLevelOperator.AdditionalPluginArguments
+        pluginInfo = pluginManager.getPluginByName(pluginName, category="TrackingExportFormats")
+
+        if pluginInfo is None:
+            logger.error("Could not find selected plugin %s", pluginName)
+            return False
+
+        plugin = pluginInfo.plugin_object
+        logger.info("Exporting tracking result using %s", pluginName)
+
+        name_format = self.topLevelOperator.getLane(lane_index).OutputFilenameFormat.value
+        partially_formatted_name = self.getPartiallyFormattedName(lane_index, name_format)
+
+        if plugin.exportsToFile:
+            filename = partially_formatted_name
+            if os.path.basename(filename) == '':
+                filename = os.path.join(filename, 'pluginExport.txt')
+        else:
+            filename = partially_formatted_name
+
+        if not filename:
+            logger.error("Cannot export from plugin with empty output filename")
+            return False
+
+        self.progressSignal(-1)
+
+        status = self._doPluginExport(lane_index, filename, plugin, checkOverwriteFiles, argsSlot)
+
+        self.progressSignal(100)
+
+        if not status:
+            return False
+
+        logger.info("Export done")
+        return True
+
+    def post_process_lane_export(self, lane_index: int, checkOverwriteFiles: bool = False):
+        # `checkOverwriteFiles` parameter ensures we check only once for files that could be overwritten, pop up
+        # the MessageBox and then don't export. For the next round we click the export button,
+        # we really want it to export, so checkOverwriteFiles=False.
+        # Plugin export if selected
+        op = self.topLevelOperator
+
+        logger.info("Export source is: %s", op.SelectedExportSource.value)
+        plugin_export_selected = op.SelectedExportSource.value == OpTrackingBaseDataExport.PluginOnlyName
+
+        if plugin_export_selected:
+            logger.info("Export source plugin selected!")
+            return self._export_with_plugin(lane_index, checkOverwriteFiles, op.SelectedPlugin.value)
+
+        return True
+
+    def getPartiallyFormattedName(self, lane_index: int, path_format_string: str) -> str:
+        ''' Takes the format string for the output file, fills in the most important placeholders, and returns it '''
+
+        raw_dataset_info = self.topLevelOperator.RawDatasetInfo[lane_index].value
+        project_path = self.topLevelOperator.WorkingDirectory.value
+        dataset_dir = PathComponents(raw_dataset_info.filePath).externalDirectory
+        abs_dataset_dir = make_absolute(dataset_dir, cwd=project_path)
+
+        nickname = raw_dataset_info.nickname.replace('*', '')
+        if os.path.pathsep in nickname:
+            nickname = PathComponents(nickname.split(os.path.pathsep)[0]).fileNameBase
```

Comment:
I am not sure, but:
```python
nickname = pathlib.Path(nickname).stem
```

Reply:
Not in this case.
If we have `dataset.hdf5/raw` as an input `stem` function will return `raw` instead of `dataset.hdf5`.
I think in the future we should split external/internal paths